### PR TITLE
Updated main.cpp (GCI-2018)

### PR DESCRIPTION
**Pull request by D3ezy27**

### CODE REVIEW

File: demo/ApproxAligner/main.cpp [hunk 1]
```diff
@@ -1,13 +1,16 @@
 #include "generate_approx_timestamp.h"
+#include <iostream>
+using namespace std;
 
 int main()
 {
-	std::cout<<"Enter path to the subtitle file : ";
+cout<<"Enter path to the subtitle file : ";
```
Comment (harrynull) [inline demo/ApproxAligner/main.cpp]: Please notice your indent. Also, using `using namespace std;` usually doesn't consider as a good practice.

Comment (canihavesomecoffee) [reply demo/ApproxAligner/main.cpp]: I agree with both remarks.

File: demo/ApproxAligner/main.cpp [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
-    std::string filename;
-    std::cin>>filename;
+string fileName;
```
Comment (canihavesomecoffee) [inline demo/ApproxAligner/main.cpp]: Indentation is off.

File: demo/ApproxAligner/main.cpp [hunk 3]
```diff
@@ -1,13 +1,16 @@
 #include "generate_approx_timestamp.h"
+#include <iostream>
```
Comment (canihavesomecoffee) [inline demo/ApproxAligner/main.cpp]: Can you explain why this include is necessary?

**Comment by saurabhshri** [comment]
Except the change of `filename` to `fileName`, none of the other ones are really needed (as explained in the review comments). Please update your PR with the same. Thanks!

**Comment by canihavesomecoffee** [comment]
Closing as there's no more activity on this. If you want to conintue, please open a new one, keeping the changes requested in mind.

**Pull request closed without merging**